presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -90,6 +90,94 @@ Additionally, the relations within a ``WITH`` clause can chain::
     relation is used. This means that if the relation is used more than once and the query
     is non-deterministic, the results may be different each time.
 
+SELECT Clause
+-------------
+
+The ``SELECT`` clause specifies the output of the query. Each ``select_expression``
+defines a column or columns to be included in the result.
+
+.. code-block:: none
+
+    SELECT [ ALL | DISTINCT ] select_expression [, ...]
+
+The ``ALL`` and ``DISTINCT`` quantifiers determine whether duplicate rows
+are included in the result set. If the argument ``ALL`` is specified,
+all rows are included. If the argument ``DISTINCT`` is specified, only unique
+rows are included in the result set. If neither is specified, the behavior
+defaults to ``ALL``.
+
+**Select expressions**
+
+Each ``select_expression`` must be in one of the following forms:
+
+.. code-block:: none
+
+    expression [ [ AS ] column_alias ]
+
+.. code-block:: none
+
+    expression.* [ [ AS ] ( column_alias [, ...] ) ]
```

Comment:
I might break this into two elements, one for relations (without the column alias option) and one for row expressions.  I think this will help people understand, with a quick glance, that `.*` and column aliases are is only allowed on row expressions, and that `.*` is allowed for relations. Something like this:
```
.. code-block:: none

     row_expression.* [ [ AS ] ( column_alias [, ...] ) ]

.. code-block:: none

     relation.*
```